### PR TITLE
{perf}[intel/2017.02] IMB v4.1

### PR DESCRIPTION
**Pull request by henkela**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/i/IMB/IMB-4.1-intel-2017.02.eb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+description = """The Intel MPI Benchmarks perform a set of MPI performance measurements for point-to-point and
+ global communication operations for a range of message sizes"""
+
+toolchain = {'name': 'intel', 'version': '2017.02'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://software.intel.com/sites/default/files/managed/34/aa/']
+sources = ['%(name)s_%(version)s.tgz']
+
+prebuildopts = "cd src && "
```
Comment (boegel) [inline easybuild/easyconfigs/i/IMB/IMB-4.1-intel-2017.02.eb]: use `start_dir = 'src'` like we are in existing IMB easyconfigs?

File: easybuild/easyconfigs/i/IMB/IMB-4.1-intel-2017.02.eb [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+buildopts = "all"
```
Comment (boegel) [inline easybuild/easyconfigs/i/IMB/IMB-4.1-intel-2017.02.eb]: is this strictly required? it's not there in #4517...

Is that why `IMB-IO` is being listed here in the sanity check, but not in #4517? If so, we should use `all` there too and enhance the sanity check?

File: easybuild/easyconfigs/i/IMB/IMB-4.1-intel-2017.02.eb [hunk 3]
```diff
@@ -0,0 +1,55 @@
+# Built with EasyBuild version 3.1.1 on 2017-04-21_13-11-20
```
Comment (boegel) [inline easybuild/easyconfigs/i/IMB/IMB-4.1-intel-2017.02.eb]: please remove this comment

File: easybuild/easyconfigs/i/IMB/IMB-4.1-intel-2017.02.eb [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+files_to_copy = [(['src/IMB-*'], 'bin')]
+
+sanity_check_paths = {
+    'files': ['bin/IMB-EXT', 'bin/IMB-IO', 'bin/IMB-MPI1', 'bin/IMB-NBC', 'bin/IMB-RMA'],
+    'dirs': [],
+}
+
+moduleclass = 'perf'
+
+# Build statistics
```
Comment (boegel) [inline easybuild/easyconfigs/i/IMB/IMB-4.1-intel-2017.02.eb]: please remove this block (we should make `--new-pr` take care of this actually, see https://github.com/hpcugent/easybuild-framework/issues/1674

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2066.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e7752ddbe0ee755f2246a7c9a7aee689 for a full test report.

**Comment by henkela** [comment]
Added this one to #4517

**Pull request closed without merging**